docs/rules/require-expect.md

```diff
@@ -0,0 +1,123 @@
+# Ensure that `expect` is called
+
+QUnit's `assert.expect(...)` helps developers create tests that correctly fail
+when their expected number of assertions are not called. QUnit will throw an
+error if no assertions are called by the time the test ends unless a developer
+also calls `assert.expect(0)`. QUnit also has a [configuration option to require
+`expect` for every test](https://api.qunitjs.com/QUnit.config/).
+
+This rule checks for `expect` at linting time. The default "always" option
+requires that `expect` is called in each test. The "except-simple" option only
+requires an `expect` call when an assertion is called inside of a block or when
+`assert` is passed to another function. The rational here is that by wrapping
+`assert` statements with logic, developers are at risk of creating tests that
+incorrectly pass by skipping assertions.
+
+# Rule Details
+
+When using the default "always" option, each test needs an expect call. So this
+example is not valid.
+
+```js
+test('name', function(assert) {
+    assert.ok(true);
+});
+```
+
+This example would not warn.
+
+```js
+test('name', function(assert) {
+    assert.expect(0);
```

Comment:
Maybe `assert.expect(1);`, since there's an assertion right below it? (I understand that this won't lint, but the obvious runtime problem might confuse readers.)